server/backend/quiz/serializers.py

```
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import Category, Question, Option, QuizAttempt, UserAnswer
```

Ruff (0.8.2)

`.models.UserAnswer` imported but unused

Remove unused import: `.models.UserAnswer`

(F401)